pkg/server/container_create_windows.go

```diff
@@ -324,15 +324,27 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 				return nil, errors.Errorf(`pipe mount.HostPath '%s' not supported for LCOW`, formattedSource)
 			}
 		} else {
-			s, err := os.Stat(formattedSource)
+			s, err := c.os.Stat(formattedSource)
 			if err != nil {
 				return nil, errors.Wrapf(err, "failed to Stat mount.HostPath '%s'", formattedSource)
 			}
 			if !s.IsDir() {
 				ext := strings.ToLower(filepath.Ext(formattedSource))
 				if ext == ".vhd" || ext == ".vhdx" {
 					mo.Type = "virtual-disk"
-					mo.Options = append(mo.Options, "bind")
+				}
+			} else {
+				if formattedDestination == "/tmp" && c.config.PluginConfig.TmpTemplateVhdPath != "" {
```

Comment:
I think we should look for some special value at the start of `formattedSource` to determine if we take this code path. For instance, they could set source to something like `templatevhd://G:\`. Then we won't break the ability to explicitly map a directory through to `/tmp` if someone wants to do that.

Reply:
If we're sure we're reverting this code and it's definitely just a temporary workaround I'm fine with it as-is. :)

Reply:
Its for sure getting reverted.